pkg/libvirttools/virtualization.go

```diff
@@ -227,6 +238,21 @@ func (ds *domainSettings) createDomain(config *types.VMConfig) *libvirtxml.Domai
 	return domain
 }
 
+// Helper functions
+// TODO: set max memory and CPU to host allocatable, it is controlled by the CG anyways
```

Comment:
Can you please file an issue so we won't forget this change.

Reply:
will do. thanks.

Reply:
issue 44 is created to track this work.